advocacy_docs/edb-postgres-ai/ai-accelerator/rel_notes/src/rel_notes_1.1.0.yml

```diff
@@ -0,0 +1,17 @@
+product: AI Accelerator - Pipelines
+version: 1.1.0
+date: 1 January 2025
+intro: |
+ Updating the GA release of EDB Postgres AI - AI Accelerator - Pipelines, in response to feedback.
+highlights: |
+ - Improved API naming in functions and arguments.
+relnotes:
+- relnote: Renaming of functions/arguments
+  details: |
+    Management funtions move to use create rather than register and delete rather than drop.
```

Comment:
I would mark or highlight the words `create`, `register`, `delete`, `drop`, and `p_`. But only optional.